packages/domain-picker/src/domain-picker/index.tsx

```diff
@@ -150,9 +150,8 @@ const DomainPicker: FunctionComponent< Props > = ( {
 	const [ domainSearch, setDomainSearch ] = useState< string >( initialDomainSearch );
 	const [ domainCategory, setDomainCategory ] = useState< string | undefined >();
 
-	const domainSuggestionVendor = useSelect( ( select ) =>
-		select( DOMAIN_SUGGESTIONS_STORE ).getDomainSuggestionVendor()
-	);
+	// If no vendor given, get default vendor.
+	const domainSuggestionVendor = vendor || getDomainSuggestionsVendor();
```

Comment:
Just wandering what's the correct logic here — i.e should we use the `||` operator, or `??` ?

If we just care about `vendor` not being `undefined`, then we could even delete these lines altogether and do something like (this)[https://github.com/Automattic/wp-calypso/pull/50761/files#r592358787]

if `vendor` is an empty string, would we still want to fallback to default `getDomainSuggestionsVendor()`?

Reply:
Line removed in favour of default value in prop. https://github.com/Automattic/wp-calypso/pull/50761/commits/eebf36b5582e37d169569415cff38387eeca3410